scripts/init-repo.ps1

```diff
@@ -0,0 +1,104 @@
+<#
+.DESCRIPTION
+Initialize zure DevOps GIT repository with branches, policies, rights, build pipelines..see https://github.com/Oriflame/devops/blob/master/docs/init-repo.md
+
+.PARAMETER AzureDevOpsCollection
+Url of Azure DevOps organisation, e.g. https://dev.azure.com/organisationName/
+
+.PARAMETER DevOpsProject
+Name of the DevOps project, e.g. http://dev.azure.com/oriflame/{THIS}
+
+.PARAMETER RepositoryName
+Name of the GIT Repository, e.g. http://dev.azure.com/oriflame/something/_git/{THIS}
```

Comment:
OrganizationName is bit inconsistent - I would put there oriflame everywhere or the {organizationName} instead.
What {THIS} stands for? Again maybe use {project} and {repository} instead.

Reply:
thanks, will fix

Reply:
the {THIS} you use more often - maybe use the parameter name

**$RepositoryName**: name of the project in Azure DevOps, e.g. http://dev.azure.com/oriflame/something/_git/**{THIS}**

like

**$RepositoryName**: name of the project in Azure DevOps, e.g. http://dev.azure.com/oriflame/something/_git/**{RepositoryName}**

Reply:
fixed